x-pack/plugins/stack_alerts/public/alert_types/es_query/expression/es_query_expression.tsx

```diff
@@ -165,12 +144,14 @@ export const EsQueryExpression = ({
       <EuiTitle size="xs">
         <h5>
           <FormattedMessage
-            id="xpack.stackAlerts.esQuery.ui.selectIndex"
-            defaultMessage="Select an index and size"
+            id="xpack.stackAlerts.searchThreshold.ui.conditionPrompt"
```

Comment:
QQ, don't think this change is intended (id+message)?
![Bildschirmfoto 2022-06-27 um 09 40 36](https://user-images.githubusercontent.com/463851/175886360-9fc1fe99-140a-424b-8f4b-e6d5690f347a.png)

Reply:
@kertal "Size" control was moved down, so I updated the label too to match what we had in Alert flyout on Discover page.